docs/en/status-communities/set-up-your-communitys-approval-options.md

```diff
@@ -6,11 +6,36 @@ hide:
   - navigation
 ---
 
-# Set up your community's approval options
+# Manage community join request
 
- :octicons-tools-24: In progress
+As a community owner, you can manually approve a request from people who wants to join your community. With this option enabled, you receive a notification whenever someone requests to join your community. This way, you can decide whether to accept, reject or ignore the request.
```

Comment:
'<...> you can manually approve requests from people who want to join your community':

- 'approve requests'
- 'people who want'